### PR TITLE
Add HWA docs for Rockchip VPU (RKMPP, feat. RK3588)

### PR DESCRIPTION
**Pull request by nyanmisaka**
- Add HWA docs for Rockchip VPU (RKMPP, feat. RK3588)

### CODE REVIEW

**Comment by nyanmisaka** [comment]
> Had a quick look, this is going to have merge conflicts with #875

I can wait until after that merge and make simple changes.

**Review by felix920506 — APPROVED** [review]
Will merge near 10.9 release (mid to late April)

File: docs/general/administration/hardware-acceleration/rockchip.md [hunk 1]
```diff
@@ -0,0 +1,254 @@
+---
+uid: admin-hardware-acceleration-rockchip
+title: Rockchip VPU
+---
+
+# HWA Tutorial On Rockchip VPU
+
+This tutorial guides you on setting up full video hardware acceleration on Rockchip VPU via RKMPP.
+
```
Comment (felix920506) [inline docs/general/administration/hardware-acceleration/rockchip.md:9]: Add a line here that states Rockchip hw accel requires 10.9

File: docs/general/administration/hardware-selection.md [hunk 2]
```diff
@@ -55,7 +55,7 @@ For low power applications, Intel 12th gen or newer Atom CPUs with integrated gr
 
 :::caution SBCs (Single Board Computers)
 
-Most SBCs use low powered chipsets, often with less than ideal driver support from the chipset vendors. They are generally too slow for a good experience and/or have broken hardware acceleration support. Please avoid using SBCs such as Raspberry Pis (Including newer Raspberry Pi 5 Models) to run Jellyfin.
+Most SBCs use low powered chipsets, often with less than ideal driver support from the chipset vendors. They are generally too slow for a good experience and/or have broken hardware acceleration support. Please avoid using SBCs such as Raspberry Pis (Including newer Raspberry Pi 5 Models) to run Jellyfin. **One exception** is that the **Rockchip RK3588/3588S SoC based SBCs** provide excellent transcoding speed and power efficiency - H.264/HEVC up to 1080p@480fps or 4k@120fps transcoding with **single-digit** power consumption in watts. Another bonus is that it has **10-bit H.264 (High10) and AV1 decoding** support.
```
Comment (felix920506) [inline docs/general/administration/hardware-selection.md:58]: please disclaim that RK3588 is only supported in 10.9 or later

**Review by felix920506 — COMMENTED** [review]
I would like to start merging in some of the docs for 10.9 We are nearing the 10.9 release and merging this one won't hurt as long as it is disclaimed since it doesn't remove any 10.8 info.

File: docs/general/administration/hardware-selection.md (hunk 2 above)
Comment (felix920506) [reply docs/general/administration/hardware-selection.md:58]: I was told that the RK3588 doesn't have good software support on Linux. If that is the case then the wording should be clearer

File: docs/general/administration/hardware-acceleration/rockchip.md (hunk 1 above)
Comment (felix920506) [reply docs/general/administration/hardware-acceleration/rockchip.md:9]: Preferably an admonition warning
```suggestion
:::warning
RKMPP Hardware Acceleration requires Jellyfin 10.9 or above
:::
```

**Review by felix920506 — APPROVED** [review]

**Pull request merged**